zebra-chain/src/sapling/tree.rs

```diff
@@ -157,9 +160,36 @@ impl ZcashDeserialize for Root {
 ///
 /// Note that it's handled as a byte buffer and not a point coordinate (jubjub::Fq)
 /// because that's how the spec handles the MerkleCRH^Sapling function inputs and outputs.
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Copy)]
 struct Node([u8; 32]);
 
+impl merkle_tree::Hashable for Node {
```

Comment:
Optional: add comment explaining why this is needed (e.g. "Required to convert a NoteCommitmentTree into a SerializedTree")